app/frontend/src/components/forms/PrintOptions.vue

```diff
@@ -0,0 +1,251 @@
+<template>
+  <span>
+    <v-tooltip bottom>
+      <template #activator="{ on, attrs }">
+        <v-btn class="mx-1" @click="dialog = true" color="primary" icon v-bind="attrs" v-on="on">
+          <v-icon>print</v-icon>
+        </v-btn>
+      </template>
+      <span>Print</span>
+    </v-tooltip>
+
+    <v-dialog v-model="dialog" width="900" content-class="export-submissions-dlg">
+      <v-card>
+        <v-card-title class="headline pb-0">Printing Options</v-card-title>
+        <v-card-text>
+          <hr />
+          <p>Select a print option.</p>
+
+          <p>You can either print the page from the browser, or upload a Template file to have a structured PDF version of this submission.</p>
+          <div v-if="showTemplateUpload">
+            <v-file-input
+              counter
+              :clearable="true"
+              label="Upload template file"
+              persistent-hint
+              prepend-icon="attachment"
+              required
+              mandatory
+              show-size
+              v-model="templateForm.files"
+            />
+            <v-card-actions>
+              <v-spacer />
+              <v-tooltip top>
+                <template v-slot:activator="{ on }">
+                  <v-btn
+                    color="primary"
+                    class="btn-file-input-submit"
+                    :disabled="!templateForm.files"
+                    id="file-input-submit"
+                    :loading="loading"
+                    @click="generate"
+                    v-on="on"
+                  >
+                    <v-icon :left="$vuetify.breakpoint.smAndUp">save</v-icon>
+                    <span v-if="$vuetify.breakpoint.smAndUp">Submit</span>
```

Comment:
I'd double check if this is behaving correctly in very small viewports - I suspect this may make the entire contents disappear if viewed on mobile (which I don't think is desired).